.changeset/sour-foxes-sort.md

```diff
@@ -0,0 +1,39 @@
+---
+'@clerk/nuxt': minor
+---
+
+Add `createRouteMatcher()` helper function that allows you to protect multiple pages or API routes.
+
+For protecting pages (in a global route middleware):
+
+```ts
+// createRouteMatcher is automatically imported
+const isProtectedRoute = createRouteMatcher(['/dashboard(.*)', '/forum(.*)'])
+
+export default defineNuxtRouteMiddleware(to => {
+  const { userId } = useAuth();
+
+  if (userId.value && isProtectedRoute(to)) {
```

Comment:
Shouldn't it be `!userId.value` instead?

Reply:
good catch!